internal/learn/iota-token/staking-iota.md

```diff
@@ -0,0 +1,100 @@
+- Shimmer tokens will be distributed at a fixed rate of `1 Shimmer per 1 MIOTA per milestone`.
+
+![shimmer staking](/img/learn/iota-token/shimmer-staking.png)
+
+You can read how to stake and calculate the possible rewards [here](https://shimmer.network/claim).
+- With the launch of the Shimmer network, these generated Shimmer tokens will be distributed to the IOTA stakers. This distribution will be implemented as a functionality within the Firefly wallet. As long as the Shimmer network is not yet launched, Firefly will hold the record of rewards for the IOTA token holder. 
+
+## Staking IOTA tokens to further bootstrap the Assembly network
+
+Assembly, the smart contract implementation connecting and governing thousands of independent smart contract blockchains on top of IOTA's Tangle will have a deep connection with the IOTA tangle. The security of the Assembly network relies on the underlying security of the IOTA tangle as it solves the interconnection and exchange layer between different chains in Assembly and provides a secure anchor and immutable audit trail for state updates of IOTA smart contracts chains in Assembly.
+
+Also, the Tangle, with its newly implemented tokenization abilities and multi-asset functionalities, will be the home of all kinds of interoperable tokens. Those will be freely exchanged between accounts on different smart contract chains and accounts on the Tangle.
+
+To further reward IOTA token stakers and give them a fair share in the new Assembly network, IOTA token holders can continue to stake their IOTA tokens in order to receive Assembly tokens for 21 months after the Assembly mainnet launch.
+
+As described in the Assembly [token launch and fair distribution scheme](https://blog.assembly.sc/assembly-fair-launch-token-distribution/), a total of 20% of the Assembly supply is given to IOTA token stakers. 
+The first distribution phase happens in the 90 days as described above.
+The second ASMB distribution will happen during the first 21 months of the newly launched Assembly network.
+
+![assembly supply](/img/learn/iota-token/assembly-supply.png)
+
+- The amount of ASMB tokens that can be distributed during these 21 months depends totally on the number of tokens that are still undistributed after the initial 90 days of staking. 
+- If less than 100% of IOTA tokens have been staked for Assembly (which would distribute 8 billion ASMB tokens to IOTA stakers), the remaining ASMB tokens will be distributed in addition to the already planned 12 billion ASMB tokens to IOTA stakers.
+If only 6 billion ASMB tokens had been distributed in the first 90 days, 14 billion ASMB tokens would still be available during the second 21 month staking period.
+- The ASMB distribution during the second staking period will happen on a **daily fixed rate**.
+- Assuming 14 billion ASMB tokens will be available to distribute, this would lead to a daily payout of: 
```

Comment:
I think this bullet point is logically part of the sub-list that follows, as it is part of the example to explain the "fixed rate" bullet point above.

Reply:
```suggestion
    - Assuming 14 billion ASMB tokens will be available to distribute, this would lead to a daily payout of: 
```